README.md

```diff
@@ -61,6 +61,20 @@ It's really that simple!
 
 For more details checkout the [docs](https://engineering.hmn.md/projects/tachyon/).
 
+## Release Process
+
+1. Create a push a new tag following the convention `vx.x.x`
```

Comment:
"Create and push"?

Reply:
This I imagine is:

- `git checkout target-branch` (most likely master)
- `git tag v1.2.3`
- `git push origin v1.2.3`

Reply:
Yeah, I don't think it's worth going into the details, unless "create and push" is technically not correct?

Reply:
I don't know if it's necessary, but it never hurts to be explicit, especially when someone has already written the docs for you 😛

Reply:
(I was pointing out a typo in the original comment)

Reply:
> (I was pointing out a typo in the original comment)

Ohhhh I see :P